### ISSUE TITLE
Error message on touch  zoom 

### ISSUE THREAD

**Issue by fanfan54** [andersevenrud/OS.js-v2#250]
An unexpected error occured, maybe a bug.

> Please report this if you think this is a bug. Include a brief description on how the error occured, and if you can; how to replicate it

I tried to use this awesome Web app on my *iPad* with *Safari* but I have accidentally zoomed on the page, and at each zoom factor change this error message pops.

```
window::onerror()
Filename: http://osjsv2.0o.no/packages/extras/GoogleMail/combined.js
Line: 1108:36
Message: TypeError: undefined is not an object (evaluating 'gapi.client.gmail.users')
```

**Comment by andersevenrud** [comment]
This error comes from Google APIs apparently.

The Google Mail application really needs some love, but I have been busy with other stuff. There are also some other problems that will be fixed down the line #216

Thanks for reporting! :)

**Comment by andersevenrud** [comment]
This is also related to #112
